sdks/python/apache_beam/runners/portability/fn_api_runner/fn_runner.py

```diff
@@ -78,12 +78,12 @@
 from apache_beam.runners import pipeline_context
 from apache_beam.runners import runner
 from apache_beam.runners.portability import artifact_service
-from apache_beam.runners.portability import fn_api_runner_transforms
 from apache_beam.runners.portability import portable_metrics
-from apache_beam.runners.portability.fn_api_runner_transforms import create_buffer_id
-from apache_beam.runners.portability.fn_api_runner_transforms import only_element
-from apache_beam.runners.portability.fn_api_runner_transforms import split_buffer_id
-from apache_beam.runners.portability.fn_api_runner_transforms import unique_name
+from apache_beam.runners.portability.fn_api_runner import transforms
```

Comment:
Seeing this unqualified makes me realize that it's ambiguous with the notion of a PTransform (and the apache_beam.transforms package). Maybe we should call it `optimizations` or `translations`?

Reply:
Done.